fix(provider): render longer provider names, make them copiable

jzsfkzm opened the pull request:
closes #858

jzsfkzm committed:
fix(provider): render longer provider names, make them copiable

closes #858

baktun14 reviewed (dismissed):
Nice!🔥 Can you just post a quick screenshot of how it looks like please?

jzsfkzm committed:
fix(deployment): also use CopyTextToClipboardButton on URIs

refs #858

jzsfkzm commented:
> Nice!🔥 Can you just post a quick screenshot of how it looks like please?

Thanks, sure, here you are @baktun14. Somewhat played around with col widths on this table, all those had the same width, now some columns are wider:
<img width="1296" alt="Screenshot 2025-02-24 at 0 18 03" src="https://github.com/user-attachments/assets/885ab409-bf8a-4cdc-8e8b-f2839c844469" />

Added it here too:
<img width="1288" alt="Screenshot 2025-02-24 at 0 19 01" src="https://github.com/user-attachments/assets/0185d0bb-08dc-4609-a681-aac5fc7cff01" />

And here as well, also changed the icon next to URIs:
<img width="637" alt="Screenshot 2025-02-24 at 0 20 56" src="https://github.com/user-attachments/assets/023f6e1c-835d-431f-b2e8-ff23f6d8d5dc" />
